app/views/outings/index.html.erb

```diff
@@ -0,0 +1,12 @@
+<div class='container'>
+
+    <% @outings.each do |outing|%>
```

Comment:
@Dpalazzari This enumerable we could break out into a partial and re-use in the per-politician(category) index view as well. @drod1000 will take care of breaking it out into a partial once he uses this piece in his implementation. FYI @njgheorghita